build/templates/flutter-install.yml

```diff
@@ -10,9 +10,3 @@ steps:
       mode: 'auto'
       channel: 'stable'
       version: 'latest'
```

Comment:
Not related to this PR, but shouldn't this version be fixed?

Reply:
We should probably lock the major.

Reply:
Here is the documentation, I think it needs to be fixed if we specify something, it can be 3.X.X. https://marketplace.visualstudio.com/items?itemName=Hey24sheep.flutter

Reply:
Yep, you can take note of if to fix it in a future PR.